test.txt

```diff
@@ -1,4 +1,7 @@
 Hello there
 Lets try SSH
 
+<<<<<<< HEAD
```

Comment:
Could you remove these conflict lines? `<<<<<<< HEAD`

Also in the bottom `>>>>>>> upstream/master`